client/my-sites/plugins/plugin-activate-toggle/test/index.jsx

```diff
@@ -36,48 +37,35 @@ describe( 'PluginActivateToggle', function() {
 		} );
 		mockery.registerSubstitute( 'matches-selector', 'component-matches-selector' );
 		mockery.registerSubstitute( 'query', 'component-query' );
```

Comment:
We can circle back to this later, but mocking the component directly using these dependencies is a bit cleaner and reduces the duration of the test by about ~30-40% in my testing:

```diff
diff --git a/client/my-sites/plugins/plugin-activate-toggle/test/index.jsx b/client/my-sites/plugins/plugin-activate-toggle/test/index.jsx
index c5a9bc8..e9034bd 100644
--- a/client/my-sites/plugins/plugin-activate-toggle/test/index.jsx
+++ b/client/my-sites/plugins/plugin-activate-toggle/test/index.jsx
@@ -5,7 +5,6 @@ import { expect } from 'chai';
 import identity from 'lodash/identity';
 import mockery from 'mockery';
 import { mount } from 'enzyme';
-import noop from 'lodash/noop';
 import React from 'react';
 import { spy } from 'sinon';
 
@@ -15,6 +14,7 @@ import { spy } from 'sinon';
 import fixtures from './fixtures';
 import mockedActions from './mocks/actions';
 import mockedPluginAction from './mocks/plugin-action';
+import EmptyComponent from 'test/helpers/react/empty-component';
 import useFakeDom from 'test/helpers/use-fake-dom';
 import useMockery from 'test/helpers/use-mockery';
 
@@ -32,11 +32,7 @@ describe( 'PluginActivateToggle', function() {
                mockery.registerMock( 'analytics', analyticsMock );
                mockery.registerMock( 'my-sites/plugins/plugin-action/plugin-action', mockedPluginAction );
                mockery.registerMock( 'lib/plugins/actions', mockedActions );
-               mockery.registerMock( 'component-classes', function() {
-                       return { add: noop, toggle: noop, remove: noop }
-               } );
-               mockery.registerSubstitute( 'matches-selector', 'component-matches-selector' );
-               mockery.registerSubstitute( 'query', 'component-query' );
+               mockery.registerMock( 'my-sites/plugins/disconnect-jetpack/disconnect-jetpack-button', EmptyComponent );
 
                PluginActivateToggle = require( 'my-sites/plugins/plugin-activate-toggle' );
                PluginActivateToggle.prototype.translate = identity;
```